Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG PLEX_VER=1.18.9.2578-513b381af
-ARG PLEX_SHA=8854a6aebe9ac375431b2e3e5addc7b7421dea4a
+ARG PLEX_VER=1.19.1.2589-2de7f3266
+ARG PLEX_SHA=2b7a08c41b99b40a3cd45cf687ffd3de3c61c60c
 ARG XMLSTAR_VER=1.6.1
 ARG CURL_VER=curl-7_66_0
```

Comment:
Can you bump this to 7.68 please

Reply:
You do not want 7.69.1? https://curl.haxx.se/changes.html#7_69_1

Reply:
Yeah sure, I'd be willing to try it. Last time we bumped curl though, it didn't end well. There was random breakage with Plex being able to request stuff

Reply:
https://abi-laboratory.pro/index.php?view=timeline&l=curl

Reply:
Awesome, that's good by me then. It was likely an ABI incompatibility, given that we were replacing libs without relinking